agent/mibgroup/disman/ping/pingCtlTable.c

```diff
@@ -4223,7 +4230,7 @@ write_pingCtlRowStatus(int action,
             vars = NULL;
 
             /*
-             * ��nameΪ�յ����������ֶμӵ�var�����б���ĩβ 
+             * 将name为空的三个索引字段加到var变量列表的末尾 
```

Comment:
This looks like an unintended change?

Reply:
You're right. My guess is that the Github web editor added that. I'll figure out a way to remove it.

Reply:
Should be good now. Thanks for catching that accidental modification.